heareval/tasks/dcase2016_task2.py

```diff
@@ -44,7 +48,29 @@
     "splits": [
         {"name": "train", "max_files": 10},
         {"name": "test", "max_files": 10},
+        {"name": "valid", "max_files": 10},
```

Comment:
Are these the actual split max_files that we are going to use during evaluation? Have we set the final values for these yet for all the open tasks or are we still in dev mode for this?

Reply:
These are still in dev.

Reply:
@khumairraj Can we just set max_files to None in the non-small case, so I can switch to doing full preprocessing of the open tasks?

Reply:
Will handle in a different pr as discussed.